applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -295,6 +323,21 @@ def GetNodalGradient(self, variable):
             gradient[node.Id] = node.GetSolutionStepValue(variable)
         return gradient
 
+    def GetElementalGradient(self, variable):
```

Comment:
I think this method should also be added to the `ResponseFunctionInterface` with a `NotImplementedError`

Reply:
i just see its already there.